pkg/controllers/secretsync/secret_sync_controller.go

```diff
@@ -105,13 +104,11 @@ func (r *UserDataSecretController) syncSecretData(ctx context.Context, source *c
 		return fmt.Errorf("source secret does not have user data")
 	}
 
-	formatEncoded := base64.StdEncoding.EncodeToString([]byte("ignition"))
-
 	target.SetName(managedUserDataSecretName)
 	target.SetNamespace(r.ManagedNamespace)
 	target.Data = map[string][]byte{
 		"value":  userData,
-		"format": []byte(formatEncoded),
+		"format": []byte("ignition"),
```

Comment:
Does this not need to be base64-encoded anymore?

Reply:
Correct, this was a case of double base64 encoding. This fixes it.